BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Jimmy/JimmySecondViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  JimmySecondViewController.swift
+//  BackToBasicsUIKit
+//
+//  Created by Jimmy Gutierrez on 5/23/19.
+//  Copyright © 2019 SwiftMiami. All rights reserved.
+//
+
+import UIKit
+
+protocol canRecieve {
```

Comment:
I would add that even then I find this name wrong. The typical naming convention for something like this is `{name of the VC}Delegate`. In this case it would be: `protocol JimmySecondViewControllerDelegate: class {`

While delegates use protocols they have a specific use case and conventions. When you work in teams following platform and community conventions is super important.